content/en/about-the-docs/_index.md

```diff
@@ -59,4 +99,20 @@ From Corda 5.0 onwards, there is one documentation set, but the features that re
 >}}
 
 * **A**: The heading without the 'Enterprise only' icon is the most common and it is used where the content applies to all Corda users.
-* **B**: The ‘Enterprise only' icon is used mostly in headings, indicating that the content under that heading relates to a Corda Enterprise feature. On occasions, the icon may be used inline to identify sentences, paragraphs, or tables that apply to a Corda Enterprise feature.
\ No newline at end of file
+* **B**: The ‘Enterprise only' icon is used mostly in headings, indicating that the content under that heading relates to a Corda Enterprise feature. On occasions, the icon may be used inline to identify sentences, paragraphs, or tables that apply to a Corda Enterprise feature.
+
+## Reporting an Error in the Docs
+The R3 Technical Writing team continuously strive to improve our documentation and overall content strategy. As we make improvements, we would love to get your feedback on what we are doing well, and what we could be doing better. 
+
+Click [here](mailto:docs@r3.com) to email the R3 Technical Writing team. To help us deal with your feedback effectively, please include all of the following:
```

Comment:
```suggestion
Use the [docs@r3.com](mailto:docs@r3.com) email address to contact the R3 Technical Writing team. To help us deal with your feedback effectively, please include all of the following:
```
Vague link phrasing affects usability and is not very informative. Avoid using "Click here" phrases in the docs.

Reply:
I agree